x11-misc/polybar/metadata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>kajzersoze72@gmail.com</email>
+		<name>Kajzer</name>
+	</maintainer>
+	<maintainer type="project">
```

Comment:
Are you sure they are willing to co-maintain it?

Reply:
You mean "Gentoo Desktop Miscellaneous Project" ?
No, I'm not sure, I thought I would just add that, should I remove it ?

Reply:
I think it's better to ask them :)

Reply:
I removed it :)